docsrc/community.rst

```diff
@@ -3,51 +3,69 @@
+Below is a description of what these media are used for.
 
-Mailing list
-------------
+Mailing lists
+-------------
 
-The GeoNetwork mailing lists provide a place for ongoing discussion about the project.
-Users are invited to submit questions to the users mailing lists.
+The GeoNetwork mailing lists provide a place for users to discuss and share their experience.
 
-Find instructions on signing up for the users or developers lists on the pages
+Please note: Mailing lists are **not** intended for support.
+See next section for a way to get support.
+
+Find instructions on signing up for the userslists on the pages
 below:
 
-* For users:
+* GeoNetwork Users List: https://lists.sourceforge.net/lists/listinfo/geonetwork-users
 
- * GeoNetwork Users List: https://lists.sourceforge.net/lists/listinfo/geonetwork-users
+* Liste d'utilisateurs GeoNetwork francophone: https://lists.sourceforge.net/lists/listinfo/geonetwork-users-fr
 
- * Liste d'utilisateurs GeoNetwork francophone: https://lists.sourceforge.net/lists/listinfo/geonetwork-users-fr
+* Lista de Usuarios de GeoNetwork español: https://lists.sourceforge.net/lists/listinfo/geonetwork-usuarios-es
 
- * Lista de Usuarios de GeoNetwork español: https://lists.sourceforge.net/lists/listinfo/geonetwork-usuarios-es
+.. warning::
+   Make sure you are subscribed to the mailing list prior to posting, posts
+   from unsubscribed people are rejected with no notification sent to the
+   mailing list administrator.
 
-* For developers:
+GIS StackExchange
+-----------------
 
- * GeoNetwork Developers List: https://lists.sourceforge.net/mailman/listinfo/geonetwork-devel
+`GIS StackExchange <https://gis.stackexchange.com/questions/tagged/geonetwork>`_ is the only official place to ask for GeoNetwork support, using the ``geonetwork`` tag.
 
- * GeoNetwork Commit List: https://lists.sourceforge.net/mailman/listinfo/geonetwork-commit
+* List of recent questions: https://gis.stackexchange.com/questions/tagged/geonetwork
 
+* List of unanswered questions: https://gis.stackexchange.com/questions/tagged/geonetwork?sort=unanswered
 
-.. warning::
-   Make sure you are subscribed to the mailing list prior to posting, posts
-   from unsubscribed people are rejected with no notification sent to the
-   mailing list administrator.
+Questions that help discover bugs should lead to the creation of bug reports in the GitHub issue tracker (see below).
+
+GitHub
+------
+
+The `GeoNetwork GitHub issues tracker <https://github.com/geonetwork/core-geonetwork/issues>`_ is where all bug reports
+and feature requests should be done.
+
+There are templates for both uses that *needs to be followed* in order to facilitate
+reproduction (for bugs) or assessment (for features).
+
+Please note that anyone creating an issue is considered responsible for maintaining activity
+on it, for example by pinging community members or providing additional information, at least
+while no contributor has been specifically assigned to it.
```

Comment:
Issues with no activity in the last 6 months will be automatically closed. If it is an issue that is really important to you, help us in looking for funding to work on it.

Reply:
Adding something like that? So people don't complaint when we close issues.

Reply:
And then add something like https://github.com/bstriner/github-bot-close-inactive-issues somewhere.